delira/data_loading/load_utils.py

```diff
@@ -1,8 +1,47 @@
 import numpy as np
+import os
+import collections
 from skimage.io import imread
 from skimage.transform import resize
 
 
+def norm_range(data):
```

Comment:
Maybe we want to add a range argument here, for being able, to use the same function for normalization between [0, 1]?

Reply:
How would you implement this feature? The `load_sample` functions gets a function, so the user has the ability to use any normalisation function.
Maybe something like that?

```python
def norm_range(mode):
    def norm_fn(data):
        # do something depending on mode
    return norm_fn
```

Reply:
I think, this is the most suitable way to implement this. I thought about additionally passing an explicit range, but this way we would not have a uniform API